History.md

```diff
@@ -1,7 +1,18 @@
 ## Master
 
-* Features
-  * Your feature goes here <Most recent on the top, like GitHub> (#Github Number)
+* Features  
+  *  Adds max_fast_inline as a configuration option for the Server object (#2406)
+
+* Bugfixes
+  * Cleanup daemonization in rc.d script (#2409)
+
+* Refactor
+  * Extract req/resp methods to new request.rb from server.rb (#2419)
+  * Refactor Reactor and Client request buffering (#2279)
+  * client.rb - remove JRuby specific 'finish' code (#2412)
+  * Consolidate fast_write calls in Server, extract early_hints assembly (#2405)
+  * Remove upstart from docs (#2408)
+  * Consolidate option handling in Server, Server small refactors, doc changes (#2389)
```

Comment:
These lines should not be here, they are already in `History.md` (https://github.com/puma/puma/pull/2406/commits/72ed95944cd7f848413f021684be5c0fa114ed9b adds them for some reason)

Reply:
This should be good now